torch/CMakeLists.txt

```diff
@@ -298,6 +298,16 @@ endif()
 
 
 add_library(torch_python SHARED ${TORCH_PYTHON_SRCS})
+
+# Use the RPATH of the linked libraries
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
Hmm, don't you want to put it under the if statement? (Because otherwise we don't want to use rpath)